tests/api/PDFDocument.spec.ts

```diff
@@ -92,6 +93,36 @@ describe(`PDFDocument`, () => {
       expect(pdfDoc).toBeInstanceOf(PDFDocument);
       expect(pdfDoc.isEncrypted).toBe(true);
     });
+
+    it(`does not throw an error for invalid PDFs when throwOnInvalidObject=false`, async () => {
+      const pdfDoc = await PDFDocument.load(invalidPdfBytes, {
+        ignoreEncryption: true,
+        parseSpeed: ParseSpeeds.Fastest,
+        throwOnInvalidObject: false,
+      });
+      expect(pdfDoc).toBeInstanceOf(PDFDocument);
+    });
+
+    it(`throws an error for invalid PDFs when throwOnInvalidObject=true`, (done) => {
```

Comment:
Jest has a way to expect a promise to reject (though it's a bit obscure). I think writing the test this way would be more straightforward:
```js
it(`throws an error for invalid PDFs when throwOnInvalidObject=true`, async () => {
  const expectedError = new Error(
    'Trying to parse invalid object: {"line":1,"column":53,"offset":13})',
  );
  await expect(
    PDFDocument.load(invalidPdfBytes, {
      ignoreEncryption: true,
      parseSpeed: ParseSpeeds.Fastest,
      throwOnInvalidObject: true,
    }),
  ).rejects.toEqual(expectedError);
});
```

Reply:
Since I wanted to test that a specific error was rejected with I used the `done` attribute but by changing the file the error became much more complex so I've reverted to using the `rejects` expectation. I didn't know I could await on this one though so good to know now.